### PR TITLE
make yaw lock of bounce efly optional and fixed typo

### PR DESCRIPTION
**Pull request by Manaball123**
## Type of change

- [ ] Bug fix
- [X] New feature

## Description

A summary of the changes along with the reasoning behind the changes.
added option to specify yaw lock for bounce efly

## Related issues

N/A

# How Has This Been Tested?

video in case no believ :D 
https://www.youtube.com/watch?v=PLe9VlmseCE
still uploading tho should be ready in a bit

# Checklist:

- [X] My code follows the style guidelines of this project.
- [X] I have added comments to my code in more complex areas.
- [X] I have tested the code in both development and production environments.

### CODE REVIEW

File: src/main/java/meteordevelopment/meteorclient/systems/modules/movement/elytrafly/modes/Bounce.java [hunk 1]
```diff
@@ -125,8 +126,20 @@ private static boolean ignoreGround(ClientPlayerEntity player) {
...
+                return mc.player.getYaw();
+            }
+            case Smart -> {
+                return Math.round((mc.player.getYaw() + 1f) / 45f) * 45f;
+            }
+            case Simple -> {
+                return elytraFly.yaw.get().floatValue();
+            }
+        };
+        throw new IllegalArgumentException("um wtf");
```
Comment (RacoonDog) [inline src/main/java/meteordevelopment/meteorclient/systems/modules/movement/elytrafly/modes/Bounce.java:141]: please replace this entire method with
```java
private float getYawDirection() {
    return switch (elytraFly.yawLockMode.get()) {
        case None -> mc.player.getYaw();
        case Smart -> Math.round((mc.player.getYaw() + 1f) / 45f) * 45f;
        case Simple -> elytraFly.yaw.get().floatValue();
    }
}
```

**Review by RacoonDog — CHANGES_REQUESTED** [review]
womp

**Comment by GoldenStack** [comment]
womp

**Comment by Manaball123** [comment]
^fixed btw

**Pull request merged**